components/ocf-host-application/TermsOfFiscalSponsorship.js

```diff
@@ -72,7 +72,7 @@ const TermsOfFiscalSponsorship = ({ checked, onChecked }) => (
         </Box>
       </Container>
     </Box>
-    <Link href="/foundation/apply/fees">
+    <Link href="/foundation/apply/new/fees">
```

Comment:
As state in other comments, if possible let's try avoiding introducing `new` in all routes. Having `fees` in the URL should be enough.

Reply:
Yes we can avoid it.